demo/apisix/conf/apisix.yaml

```diff
@@ -91,13 +91,14 @@ services:
             - - request_method
               - "~="
               - OPTIONS
-        client_id: "james_apisix"
-        client_secret: "james_apisix"
+        client_id: "james"
+        client_secret: "james"
         discovery: "http://sso.example.com/.well-known/openid-configuration"
         scope: "openid profile"
         bearer_only: true
         introspection_endpoint_auth_method: "client_secret_post"
         redirect_uri: "http://test.sso.example.com:8080/login-callback.html"
+        use_jwks: true
```

Comment:
Q:
>  "TMail web will can not SSO with that client." 

how? Is it not redirect 302 when using the web browser?
if yes, so I am just curious why it works with Krakend, (I remember the Krakend does not redirect 302...)

Reply:
This would happen if we use introspection way (oidcRPMetaDataOptionsPublic = 0, therefore I switch to jwks mode.

![Screenshot from 2023-04-25 16-54-43](https://user-images.githubusercontent.com/55171818/234521939-9335b412-2eda-4e7b-99d5-a30c0e4fc0ba.png)

> so I am just curious why it works with Krakend

Actually the Krakend in our demo use jwks mode too.